public/app/features/dashboard-scene/scene/keyboardShortcuts.ts

```diff
@@ -60,6 +61,18 @@ export function setupKeyboardShortcuts(scene: DashboardScene) {
         scene.showModal(drawer);
       }),
     });
+
+    keybindings.addBinding({
+      key: 'p s',
+      onTrigger: withFocusedPanel(scene, async (vizPanel: VizPanel) => {
+        const drawer = new ShareDrawer({
+          title: t('share-panel.drawer.share-snapshot-title', 'Share snapshot'),
+          body: new ShareSnapshot({ dashboardRef: scene.getRef(), panelRef: vizPanel.getRef() }),
```

Comment:
Not introduced in this PR but I found 2 issues with the ShareSnapshot:
* I can create a Snapshot with an empty name (the label has a `*` which seems to indicate it is a required field)
* After creation, the Cancel and Delete Snapshot buttons should be inverted (primary on the left).

Maybe we can leverage this PR to add those changes to the `ShareSnapshot` component.

Reply:
Also not introduced in this PR but related:
After creating a Snapshot, I can't go back to Snapshot creation (there is no back row button). The only way is to click "new snapshot" which doesn't sound like going back.
Shouldn't we have the back button even if it will do the same as "new snapshot"?

Reply:
@natydej thoughts on the second comment? We all agree we need to improve that part of the flow, adding (if possible) a dashboard's snapshots list and keep in the same page, but that's not part of this PR

Reply:
The idea about the name, is that it should be [prefilled], by the panel title, so it shouldn't be the case that the field is empty, the idea is to avoid that.

About the Cancel and Delete buttons, are you referring to this screen?

<img width="841" alt="Screenshot 2024-08-13 at 09 38 38" src="https://github.com/user-attachments/assets/796e0db2-29d7-4952-b8dc-97ad5d5af194">

If so, the primary is already on the left.

About after creating a new snapshot, I don't see why people want to go back, if they already has the option on creating a new one from there. Not sure if makes sense to have it there tbh. For now this is how it is:
<img width="720" alt="Screenshot 2024-08-13 at 09 41 32" src="https://github.com/user-attachments/assets/61a77e8c-f99f-4727-8739-497980078dab">

We can start on this iteration and then see if we can improve it by adding the back arrow there.

Reply:
> Not introduced in this PR but I found 2 issues with the ShareSnapshot:
> 
> * I can create a Snapshot with an empty name (the label has a `*` which seems to indicate it is a required field)
**After discussion, an empty name is something we're already supporting. I'll remove * indicator**
> * After creation, the Cancel and Delete Snapshot buttons should be inverted (primary on the left).
**This is due to `ConfirmContent` common component. I would rather keep using this component, so any change will be reflected here**